content/php-api-using-jwt/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Securing PHP API using JWT and Apache"
+date: "2021-08-4"
+coverImage: "jwt.jpg"
+author: "Harikrishna Kundariya"
+tags: ["Securing PHP API", "PHP API using JWT","PHP API using Apache"]
+Description: "Harikrishna Kundariya, a marketer, developer, IoT, ChatBot & Blockchain savvy, designer, co-founder, Director of [eSparkBiz Technologies](www.esparkinfo.com) @SaaS App Development Company. His 8+ experience enables him to provide digital solutions to new start-ups based on IoT and ChatBot. "
```

Comment:
Here, the description should be about the article, not the author. Author bio should be added separately in the pull request description as per https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md

Reply:
Done